lib/fauna/page.rb

```diff
@@ -59,8 +72,8 @@ def ==(other)
         @client == other.instance_variable_get(:@client) &&
         @set == other.instance_variable_get(:@set) &&
         @params == other.instance_variable_get(:@params) &&
-        @fauna_map == other.instance_variable_get(:@fauna_map) &&
-        @ruby_map == other.instance_variable_get(:@ruby_map)
+        @fauna_funcs == other.instance_variable_get(:@fauna_funcs) &&
```

Comment:
Well I'm personally not happy with that name, so if you have any suggestions for a better one... (though internal names don't really matter that much)